.vscode/tasks.json

```diff
@@ -137,6 +137,19 @@
                 ]
             }
         },
+        {
+            "label": "Build thermostat App (Linux)",
```

Comment:
could we add thermostat to the build_examples.py (./scripts/build/builders/host.py) and similar instead?

I am somewhat surprised this works: will gn_buiuld automatically do just thermostat?

Reply:
I was using brute force and just using the gn_build.sh script.  
I will clean these 2 comments up in a separate PR.

Reply:
I have reverted my commit that changed both launch.json and tasks.json